packages/utils/src/extension.ts

```diff
@@ -281,6 +283,27 @@ export function activate (
     stateManager.updateConfiguration('background', oldGlobalStore.bg)
   }
 
+  /**
+   * extension host has access to events
+   */
+  const env =
+   process.env.NODE_ENV === 'development' ? 'development' : 'production'
```

Comment:
Just a personal preference
```suggestion
  const env = process.env.NODE_ENV === 'development'
      ? 'development'
      : 'production'
```